pkgs/top-level/all-packages.nix

```diff
@@ -15932,6 +15932,8 @@ with pkgs;
 
   inspectrum = libsForQt5.callPackage ../applications/misc/inspectrum { };
 
+  intellij-ultimate = callPackage ../applications/editors/intellij-ultimate { };
```

Comment:
We have the whole jetbrains suite in nixpkgs already.
Try: `nix-shell -p jetbrains.idea-ultimate`

Reply:
My bad